main.go

```diff
@@ -1,44 +1,76 @@
 package main
 
 import (
+	"flag"
 	"time"
-
+	"gossip-demo/demo"
```

Comment:
Should use the full import path here `github.com/discoviking/gossip-demo/demo`.  Otherwise it doesn't compile when downloaded with `go get` as is standard.